packages/geth-rpc-gateway/main.go

```diff
@@ -0,0 +1,208 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"io"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"net/url"
+	"os"
+	"strings"
+
+	"github.com/gorilla/websocket"
+)
+
+type JSONRPCRequest struct {
+	Method string `json:"method"`
+}
+
+var upgrader = websocket.Upgrader{
+	CheckOrigin: func(r *http.Request) bool { return true },
+}
+
+var methodsUsingPrimary map[string]bool
+
+func main() {
+	// Load the target URLs from environment variables
+	primaryURLStr := os.Getenv("TARGET_URL_PRIMARY")
+	secondaryURLStr := os.Getenv("TARGET_URL_SECONDARY")
+	webSocketURLStr := os.Getenv("WEBSOCKET_TARGET_URL")
+	webSocketSecondaryURLStr := os.Getenv("WEBSOCKET_TARGET_URL_SECONDARY")
+
+	if primaryURLStr == "" || secondaryURLStr == "" || webSocketURLStr == "" || webSocketSecondaryURLStr == "" {
```

Comment:
For mainent, we don't have the `ws` endpoint for RPC anymore, so maybe we need to change this check?